k8s/eks/yaml/tg-ds-sidecar.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: testground-sidecar
+spec:
+  selector:
+    matchLabels:
+      name: testground-sidecar
+  template:
+    metadata:
+      labels:
+        name: testground-sidecar
+    spec:
+      terminationGracePeriodSeconds: 10
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      priorityClassName: system-node-critical
+      containers:
+      - name: testground-sidecar
+        image: public.ecr.aws/n6b0k8i7/testground-sidecar:eks_v1.0
+        imagePullPolicy: Always
+        command: ["testground"]
+        args: ["sidecar", "--runner", "k8s"]
+        securityContext:
+          capabilities:
+            add: ["NET_ADMIN", "SYS_ADMIN", "SYS_TIME"]
```

Comment:
Explained [here](https://github.com/testground/testground/issues/1461).

Reply:
Again, comment would be great. On top of that the `solution` part in the linked issue only explains *what* was done, not *why*. I think the latter might be more interesting to capture.

Reply:
Added a comment in the yaml.
The solution explains what was done, but the issue itself gives us an idea why that had to be added.

Reply:
I mean, why `NET_ADMIN` and `NET_RAW`? What do they allow that was previously inaccessible?

Reply:
NET_RAW was a part of a previous test; it was removed in the meantime.
So if you check the original `sidecar.yaml` that was written 2 years ago, it had these capabilities. I think these were removed by mistake, so we had to add them back.

Reply:
OK, I get that. Do you know what these capabilities enable?

Reply:
All capabilities can be found [here](https://man7.org/linux/man-pages/man7/capabilities.7.html), as the list is too big to paste it.

Reply:
Nice! Can we add this link as a comment?

Reply:
Of course, it will be added.